src/services/test-helpers/mock/mockApi.ts

```diff
@@ -28,6 +28,7 @@ import {
 	SessionIndex,
 	StakingLedger,
 } from '@polkadot/types/interfaces';
+import {} from '@polkadot/types/types';
```

Comment:
If something is used from `@polkadot/types/types`, maybe we should explicitly import that type ? Btw, I comment this line and I do not see any errors but maybe I am missing something.

Reply:
Awesome fixed, thanks so much for that last catch!